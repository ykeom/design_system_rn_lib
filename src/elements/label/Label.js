import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Label extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>label</Text>
      </View>
    );
  }
}

export default Label;
