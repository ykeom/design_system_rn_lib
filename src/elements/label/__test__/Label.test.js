import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Label from '../Label';

describe('Label', () => {
  it('should render', () => {
    const component = shallow(<Label />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
