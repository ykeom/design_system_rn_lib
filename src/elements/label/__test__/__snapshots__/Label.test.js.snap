// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    label
  </Text>
</View>
`;
